src/components/common/Loader/Loader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from  './Loader.css';
+import useStoreon from 'storeon/react'
+
+const Loader = () =>{
+	const { loading } = useStoreon('loading');
```

Comment:
I think the loader should be pure. ie Always show the loader animation. Based on the store value it would be conditionally render the loader.